blocks/rich-text/format-toolbar/index.js

```diff
@@ -225,13 +226,13 @@ class FormatToolbar extends Component {
 									onKeyPress={ stopKeyPropagation }
 								>
 									<div className="blocks-format-toolbar__link-modal-line">
-										<a
+										<ExternalLink
 											className="blocks-format-toolbar__link-value"
 											href={ formats.link.value }
-											target="_blank"
+											icon={ null }
```

Comment:
This is documented as a boolean, but we're passing as `null` here. Would `false` be most appropriate?